congress/metadata/M001200.yaml

```diff
@@ -0,0 +1,2 @@
+name: JCWilmore - Own work
+link: https://commons.wikimedia.org/wiki/User:JCWilmore
```

Comment:
This link doesn't contain the image in question. I would recommend looking for an official .gov source instead of Wikipedia anyway.